From: Linus Lüssing <linus.luessing@c0d3.blue>
Date: Mon, 18 Feb 2019 18:28:03 +0100
Subject: batctl: remove kmod-batman-adv dependency

This always pulls in the batman-adv compat 15 kernel module. However,
batctl works just as well with batman-adv-legacy (compat 14).

Signed-off-by: Linus Lüssing <linus.luessing@c0d3.blue>

diff --git a/batctl/Makefile b/batctl/Makefile
index 4e67512844d07607852a5447a336dc9f93f5990a..df96ca7325ba3f84bd682da6df32fa4489474113 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -23,7 +23,7 @@ define Package/batctl
   URL:=https://www.open-mesh.org/
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-batman-adv +libnl-tiny +libc +librt
+  DEPENDS:=+libnl-tiny +libc +librt
   TITLE:=B.A.T.M.A.N. Advanced user space configuration tool batctl
   MAINTAINER:=Simon Wunderlich <sw@simonwunderlich.de>
 endef
